felix/calc/l3_route_resolver.go

```diff
@@ -400,8 +429,8 @@ func (c *L3RouteResolver) OnHostIPUpdate(update api.Update) (_ bool) {
 		v4Addr, ok := ip.FromCalicoIP(*newCaliIP).(ip.V4Addr)
 		if ok { // Defensive; we only expect an IPv4.
```

Comment:
Hm, I need to dig into this one a bit more - surprised we filter on IPv4 here. Do we not need an IPv6 host address for IPv6 VXLAN encap?

Reply:
This is the "old" code that is registered when `c.useNodeResourceUpdates` is false (which doesn't happen for a while in recent versions, right?). We deal with the IPv6 updates in `OnResourceUpdate()`